app/views/email_change_success.scala.html

```diff
@@ -0,0 +1,36 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import uk.gov.hmrc.play.views.html.helpers.form
+
+@()(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main_template(title = messages("emailChangeSuccess.title"), bodyClasses = None) {
+
+ <h1 class="heading-large">@messages("emailChangeSuccess.heading")</h1>
+
+ <h2 class="lede" data-journey="email-address:view:change-email-success">@messages("emailChangeSuccess.headingTwo")</h2>
```

Comment:
Can you add a test to prove the GA tag is there please?
This could potentially be moved up to the `h1` just to be consistent with how it is implemented in `manage-vat-subscription-frontend`